src/components/data-management/ProjectDeleteModal.jsx

```diff
@@ -16,6 +16,7 @@ const ProjectDeleteModal = (props) => {
 
   return (
     <Modal
+      className='delete-project-modal'
```

Comment:
Why not use an id for this instead of class?

Reply:
because they are not supported for modals because you can potentially have more than one